supertokens_python/recipe/emailverification/recipe.py

```diff
@@ -239,7 +230,7 @@ def callback():
                 PostSTInitCallbacks.add_post_init_callback(callback)
 
                 return EmailVerificationRecipe.__instance
-            raise_general_exception(
+            return raise_general_exception(
```

Comment:
why do you need this return given that the type of raise_general_exception has NoReturn?

Reply:
Done